tests/python/contrib/test_ethosn/test_conv2d.py

```diff
@@ -185,6 +185,14 @@ def test_conv2d_failure():
     if not ethosn_available():
         return
 
+    _scale_error_msg = (
+        "Overall scale (of the input * weights / output) should be in the range [0, 1)"
+    )
+    if tei.get_ethosn_api_version() == 2008:
+        _scale_error_msg = (
+            "Overall scale (of the input * weights / output) should be in the range [0, 1}"
+        )
```

Comment:
Are you trying to match the error message from the Ethos-N compiler? If so, it seems to me that we should be fine as long as we could detect the error (by error code or any other form).

If you really need to check the error message for whatever reasons, we shouldn't be bothered by the small message changes. For example, you can just match `msg.startswith("Overall scale")`.